### ISSUE TITLE
Sometimes Zetta returns a 200 status code but empty response body

### ISSUE THREAD

**Issue by mdobson** [zettajs/zetta#172]
Talked with @AdamMagaluk on this. Sometimes Zetta will return a 200 OK response from a transition but no body. 

Found using the ZettaKit example client.

**Comment by mdobson** [comment]
relevant debug info from the client

```
2015-04-17 11:45:25.109 ZettaKit[22268:682110] Error: Error Domain=NSCocoaErrorDomain Code=3840 "The operation couldn’t be completed. (Cocoa error 3840.)" (No value.) UserInfo=0x7fe4495c4bd0 {NSDebugDescription=No value.}
2015-04-17 11:45:25.109 ZettaKit[22268:682110] Data:0
2015-04-17 11:45:25.110 ZettaKit[22268:682110] Resp: <NSHTTPURLResponse: 0x7fe44b892850> { URL: http://zt-1-mdobs.herokuapp.com/servers/e4def539-7f5b-4331-b89b-40fc5a67ad74/devices/098677a9-869d-4b65-acab-a2a34edee00d } { status code: 200, headers {
    "Access-Control-Allow-Origin" = "*";
    Connection = "keep-alive";
    "Content-Length" = 0;
    "Content-Type" = "application/vnd.siren+json";
    Date = "Fri, 17 Apr 2015 15:45:25 GMT";
    Server = Cowboy;
    Via = "1.1 vegur";
} }
2015-04-17 11:45:25.110 ZettaKit[22268:682110] Body: action=turn-on
2015-04-17 11:45:25.110 ZettaKit[22268:682110] Error Domain=NSCocoaErrorDomain Code=3840 "The operation couldn’t be completed. (Cocoa error 3840.)" (No value.) UserInfo=0x7fe4495c4bd0 {NSDebugDescription=No value.}
```

**Comment by AdamMagaluk** [comment]
Fixed in latest zetta@0.26.0
